src/main/java/com/redhat/red/build/finder/DistributionAnalyzer.java

```diff
@@ -133,14 +136,13 @@ private void listChildren(FileObject fo) throws IOException {
                     listChildren(layered);
                     sfs.closeFileSystem(layered.getFileSystem());
                 } catch (FileSystemException e) {
-                    LOGGER.warn("Unable to process archive/compressed file: {}", red(found));
-                    LOGGER.debug("Caught file system exception", e);
+                    LOGGER.warn("Unable to process archive/compressed file {}: {}: {}", red(found), e.getCause(), e.getMessage());
```

Comment:
We don't throw an exception here as it would prevent the creation of the entire checksum map and it's usually a harmless or expected error. I am not sure what else should be done.

Reply:
Ok. Previously though it didn't print out a stack trace unless in debug mode - why not keep that?

Reply:
I'll add it back. It doesn't look intentional.